infrastructure/swagger/swagger-fhir-generator-definitions/endpoints.yaml

```diff
@@ -10,4 +10,4 @@
 # described in https://hl7.org/fhir/r4b/http.html
 
 Device: [create, read, search]
-Product: [create, read]
+Product: [create]
```

Comment:
This either needs rebasing from the current release or you need to remove Product [create]. This is becasue this is the last step in removing the old Product. It will remove any unwanted parts of the fhir model. I would suggest rebasing as it will probably remove other parts of the code that you no longer require as part of this ticket.